pkg/secrets/check_rights.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build secrets
+
+package secrets
+
+// checkRightOptions use provide options parameters to the CheckRight function
+type checkRightOptions struct {
```

Comment:
I don't see any other options for the check rights (and this is the first change we made in a long time on this). I feel this is not needed and a simple `bool` in the `checkRights` signature would be enough and keep the code simpler.

Reply:
Yes, I was divided between simplicity and be future proof.